presto-oracle/src/main/java/io/prestosql/plugin/oracle/OracleConfig.java

```diff
@@ -67,4 +71,49 @@ public OracleConfig setNumberRoundingMode(RoundingMode numberRoundingMode)
         this.numberRoundingMode = numberRoundingMode;
         return this;
     }
+
+    @NotNull
+    public boolean isConnectionPoolEnabled()
+    {
+        return connectionPoolEnabled;
+    }
+
+    @Config("oracle.connection-pool.enabled")
+    public OracleConfig setConnectionPoolEnabled(boolean connectionPoolEnabled)
+    {
+        this.connectionPoolEnabled = connectionPoolEnabled;
+        return this;
+    }
+
+    @Min(0)
```

Comment:
nit: add tests for validation constraints

Reply:
done